lisa/operating_system.py

```diff
@@ -477,6 +488,15 @@ def _get_os_version(self) -> OsVersion:
 
         return os_version
 
+    def _update_packages(self, packages: Optional[Union[List[str]]] = None) -> None:
+        command = "DEBIAN_FRONTEND=noninteractive apt-get upgrade -y "
+        command += (
```

Comment:
use below pattern is better, avoid to join strings.

```python
        command = (
            "DEBIAN_FRONTEND=noninteractive apt-get upgrade -y "
            '-o Dpkg::Options::="--force-confdef" -o Dpkg::Options::="--force-confold"'
        )
  
```

Reply:
Fixed